Core/src/main/java/com/plotsquared/core/queue/ChunkCoordinatorBuilder.java

```diff
@@ -227,7 +240,8 @@ public ChunkCoordinatorBuilder(@NonNull ChunkCoordinatorFactory chunkCoordinator
                         this.whenDone,
                         this.throwableConsumer,
                         this.unloadAfter,
-                        this.progressSubscribers
+                        this.progressSubscribers,
+                        this.forceSync
```

Comment:
It would make sense to deprecate the existing builder and create a new method, because it's API, which now takes an additional parameter.